jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessorTest.java

```diff
@@ -837,6 +874,31 @@ public void testGetJavaContainerBuilderWithBaseImage_java12NoBaseImage()
       Assert.assertEquals(12, ex.getProjectMajorJavaVersion());
     }
   }
+  //
+  @Test
+  public void testGetPlatformsSet() throws InvalidPlatformConfigurationException {
+    Mockito.<List<? extends PlatformConfiguration>>when(rawConfiguration.getPlatforms())
+        .thenReturn(Arrays.asList(new PlatformParametersTest("amd64", "linux")));
+
+    Assert.assertEquals(
+        ImmutableSet.of(new Platform("amd64", "linux")),
+        PluginConfigurationProcessor.getPlatformsSet(rawConfiguration));
+  }
+
+  @Test
+  public void testInvalidPlatformsSet() throws InvalidPlatformConfigurationException {
+    PlatformParametersTest platform = new PlatformParametersTest(null, "linux");
+    Mockito.<List<? extends PlatformConfiguration>>when(rawConfiguration.getPlatforms())
+        .thenReturn(Arrays.asList(platform));
+
+    try {
+      PluginConfigurationProcessor.getPlatformsSet(rawConfiguration);
+      Assert.fail();
+    } catch (InvalidPlatformConfigurationException ex) {
+      Assert.assertEquals(platform.toString(), ex.getMessage());
+      Assert.assertEquals(platform.toString(), ex.getInvalidPlatform());
+    }
```

Comment:
Don't use `platform.toString()` but use an actual `String` literal to verify the returned value.